modules/op-viewing-pipeline-logs-in-kibana.adoc

```diff
@@ -0,0 +1,95 @@
+// Module included in the following assemblies:
+// cicd/pipelines/viewing-pipeline-logs-using-the-openshift-logging-operator.adoc
+//
+
+[id="op-viewing-pipeline-logs-in-kibana_{context}"]
+= Viewing pipeline logs in Kibana
+
+.Procedure
+
+To view pipeline logs in the Kibana web console:
+
+. Log in to *Red Hat OpenShift Container Platform* as a cluster administrator, or a user with equivalent permissions.
+.. In the top right of the menu bar, click the *grid* icon → *Observability* → *Logging*. The Kibana web console is displayed.
+
+. On the left navigation panel of the *Kibana* web console, click *Management*.
+.. Click *Create index pattern*.
+.. Under *Step 1 of 2: Define index pattern* → *Index pattern*, enter a *`pass:[*]`* pattern and click *Next Step*.
+.. Under *Step 2 of 2: Configure settings* → *Time filter field name*, select *@timestamp* from the drop-down menu, and click *Create index pattern*.
+
+. On the left navigation panel of the *Kibana* web console, click *Discover*.
+.. Click *Add a filter +* → *Edit Query DSL*.
++
+[NOTE]
+====
+* For each of the example filters that follows, edit the query and click *Save*.
+* The filters are applied one after another.
+====
++
+... Filter the containers related to pipelines:
++
+.Example query to filter pipelines containers
+[source,json]
+----
+{
+  "query": {
+	"match": {
+  	"kubernetes.flat_labels": {
+    	"query": "app_kubernetes_io/managed-by=tekton-pipelines",
+    	"type": "phrase"
+  	}
+	}
+  }
+}
+----
++
+... Filter all containers that are _not_ `place-tools` container. As an illustration of using the graphical drop-down menus instead of editing the query DSL, consider the following approach:
```

Comment:
```suggestion
... Filter all containers that are not `place-tools` container. As an illustration of using the graphical drop-down menus instead of editing the query DSL, consider the following approach:
```

I don't think _emphasis_ is needed here